rpc/wrtc_signaling_answerer.go

```diff
@@ -238,6 +245,14 @@ func (ans *webrtcSignalingAnswerer) Stop() {
 // the designated WebRTC data channel is passed off to the underlying Server which
 // is then used as the server end of a gRPC connection.
 func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_AnswerClient) (err error) {
+	// Maintain and eventually log a collection of stats for each answering attempt.
+	stats := &answererStats{}
+	defer func() {
+		if ans.logStats {
+			stats.log(ans.logger)
```

Comment:
For some reason one answerer is outputting `Connection establishment succeeded` and the other is outputting `Connection establishment failed` both with similar stats for a single connection establishment. Looking into that before I pull this out of "draft"; makes me think I'm misunderstanding how the two answerers interact.

Reply:
I believe this was due to RSDK-7311 and should be fixed by #283 from my testing. `app`'s goutils would have to have #283 in order and `rdk` would have to have this PR's changes to have a single `Connection establishment [succeeded|failed]` message from a running RDK. Curious if we want to wait for `app` to upgrade its `goutils` version to one that contains #283 (if reviewers agree on merging that PR) before merging this PR/upgrading the `goutils` version in `rdk`, @cheukt.

Reply:
would wait until #283 is mergedbefore upgrading goutils in rdk

Reply:
Will move this PR out of draft but won't merge until we fix the double `AnswerRequest_Init` issue.

Reply:
Confirmed since `app` has been upgraded to have #283, the double `AnswerRequest_Init` issue is gone.